src/test/java/com/qualimente/training/addressbook/ControllerTest.java

```diff
@@ -36,7 +43,7 @@ public void getAddressesForCustomer_should_find_customer_addresses_in_repository
     addressesByCustomerId.put(customerId, expectedAddresses);
     AddressDAO AddressDAO = new AddressDAOMemoryImpl(addressesByCustomerId);
 
-    Controller controller = new Controller(AddressDAO);
+    Controller controller = new Controller(AddressDAO, new LocationDataValidator());
```

Comment:
For consistency, consider constructing the LocationDataValidator in the same manner as the AddressDAO.